unobstructed_lanechange/config/parameters.yaml

```diff
@@ -67,6 +67,20 @@ enable_object_avoidance_lc : false
 #Units: seconds
 min_timestep : 0.1
 
+<<<<<<< HEAD
```

Comment:
Whoa, careful with these merge conflict resolutions. Definitely want to make sure we clean this up, looks like the unresolved conflict got committed and pushed. I'm guessing we want to take the HEAD branch here, but take a look at it please.